software/bsg_manycore_lib/bsg_tile_group_barrier.hpp

```diff
@@ -40,6 +40,22 @@ template <int BARRIER_X_DIM>
 class bsg_row_barrier {
 private:
 
+        // The initialization assignments here have no effect if the
+        // barrier is instantiated at global scope (as is recommended
+        // practice)
+
+        // When the barrier is declared at global scope _local_alert
+        // and _done_list are located in the .bss.dmem segment.
```

Comment:
Section not segment. Two different things. Segments don’t have names.

Reply:
fx'd